pkgs/development/compilers/graalvm/graalvm-oracle/hashes.nix

```diff
@@ -0,0 +1,42 @@
+{
+  "22" = {
+    "hashes" = {
+      "aarch64-linux" = {
+        sha256 = "d303d30c10764feaab076efe79019df3c981b0487fcac3df7a5313f76f61040e";
+        url = "https://download.oracle.com/graalvm/22/archive/graalvm-jdk-22_linux-aarch64_bin.tar.gz";
+      };
+      "x86_64-linux" = {
+        sha256 = "d1860b5b7588310e70b259c891156f6d0cbc34d0d1feec3b37169ed2a415f3c3";
```

Comment:
I think these hashes are correct, I calculated them with nix-prefetch-url but I need to double check.